extension/.env.example

```diff
@@ -1,5 +1,5 @@
 PRIVATE_KEY_STORAGE_KEY=default-private-key
-AGGREGATOR_URL=http://localhost:3000
-DEFAULT_CHAIN_ID=31337
+AGGREGATOR_URL=https://arbitrum-testnet.blswallet.org/
```

Comment:
before merging @voltrevo should update the public aggregator running on this endpoint to use [this](https://github.com/web3well/bls-wallet/blob/new-deployment/contracts/networks/arbitrum-testnet-goerli.json) config